hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestFromClientSide5.java

```diff
@@ -2274,6 +2274,20 @@ public void testCellSizeLimit() throws IOException {
         // expected
       }
     }
+    //test cell size no limit
+    final TableName noLimitTable = TableName.valueOf("testCellSizeNoLimit");
+    TableDescriptorBuilder.ModifyableTableDescriptor noLimitTableDescriptor =
+      new TableDescriptorBuilder.ModifyableTableDescriptor(noLimitTable)
+        .setValue(HRegion.HBASE_MAX_CELL_SIZE_KEY, Integer.toString(0));
+    noLimitTableDescriptor.setColumnFamily(familyDescriptor);
+    try (Admin admin = TEST_UTIL.getAdmin()) {
+      admin.createTable(noLimitTableDescriptor);
+    }
+    // Will succeed
+    try (Table nt = TEST_UTIL.getConnection().getTable(noLimitTable)) {
+      nt.put(new Put(ROW).addColumn(FAMILY, QUALIFIER,  new byte[9 * 1024 * 1024]));
```

Comment:
NIT: For clarity, can we use `HRegion.DEFAULT_MAX_CELL_SIZE + 1`, instead?

Reply:
Since the client will also check hbase.client.keyvalue.maxsize, i use HRegion.DEFAULT_MAX_CELL_SIZE - 1024 and 1024 + 1, instead.